neuralmonkey/runners/rnn_runner.py

```diff
@@ -241,19 +241,19 @@ def next_to_execute(self) -> NextExecute:
             raise Exception(
                 "Nothing to execute, if there is already a result.")
 
-        to_run = {'logprobs': self._decoder.train_logprobs[self._time_step]}
+        to_run = {'logprobs': self._decoder.runtime_logprobs[self._time_step]}
 
         self._current_beam_batch = self._to_exapand.pop()
 
         if self._current_beam_batch is not None:
             additional_feed_dict = {t: index for t, index in zip(
-                self._decoder.train_inputs[1:],
+                tf.unpack(self._decoder.train_inputs[1:]),
```

Comment:
Když se tady feeduje do `train_inputs`, tak to ale přece vůbec neovlivní ty `runtime_logprobs`, ty závísí jenom na loop function, takže nám pak vždycky dají distribuci, která odpovídá greedy dekódování.